health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/hooks/useFileDownload.js

```
@@ -0,0 +1,10 @@
+import { useQuery } from "react-query";
+import SearchFileIds from "./services/SearchFileIds";
+
+const useFileDownload = (data, config = {}) => {
+  debugger;
```

Biome

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)